src/main/java/org/mule/extension/api/introspection/ImmutableOperationModel.java

```diff
@@ -19,18 +20,19 @@
 public class ImmutableOperationModel extends AbstractParameterizedModel implements OperationModel
```

Comment:
looks like the totality of changes in this class are either neutral or unnecesary